dashboards/management/commands/data_quality.py

```diff
@@ -0,0 +1,360 @@
+    for b in bills_with_votes_with_voters:
+        for vote_object in b.votes.all():
+            total_yes = 0
+            total_no = 0
+            total_absent = 0
+            total_excused = 0
+            total_not_voting = 0
+            total_other = 0
+
+            voter_count_yes = 0
+            voter_count_no = 0
+            voter_count_absent = 0
+            voter_count_excused = 0
+            voter_not_voting = 0
+            voter_count_other = 0
+
+            vote_counts = vote_object.counts.all()
+            votes = vote_object.votes.all()
+            # Parsing through vote_counts
+            for count in vote_counts:
+                if "yes" == count.option:
+                    total_yes = count.value
+                elif "no" == count.option:
+                    total_no = count.value
+                elif "absent" == count.option:
+                    total_absent = count.value
+                elif "excused" == count.option:
+                    total_excused = count.value
+                elif "not voting" == count.option:
+                    total_not_voting = count.value
+                elif "other" == count.option:
+                    total_other = count.value
+                else:
+                    print("Other option found in vote_counts: ", count)
+                    print(count.__dict__)
+            # Parsing through voters and adding up their votes
+            for voter in votes:
+                if voter.option == "yes":
+                    voter_count_yes += 1
+                elif voter.option == "no":
+                    voter_count_no += 1
+                elif voter.option == "absent":
+                    voter_count_absent += 1
+                elif voter.option == "excused":
+                    voter_count_excused += 1
+                elif voter.option == "abstain":
+                    voter_not_voting += 1
+                elif voter.option == "other":
+                    voter_count_other += 1
+                else:
+                    print(voter.option)
+            # Checking to see if votes and vote counts match
+            if (voter_count_yes != 0 and voter_count_yes != total_yes) or (voter_count_no != 0 and voter_count_no != total_no):
+                total_votes_where_votes_dont_match_voters += 1
+    bill_vote_data = {
+        "total_votes_without_voters": total_votes_without_voters,
+        "total_votes_where_votes_dont_match_voters": total_votes_where_votes_dont_match_voters
+    }
+
+    return bill_vote_data
+
+
+def write_json_to_file(filename, data):
+    with open(filename, "w") as file:
+        file.write(data)
+
+# Example command
+# docker-compose run --rm django poetry run ./manage.py data_quality Virginia
+class Command(BaseCommand):
+    help = "export data quality as a json"
+
+    def add_arguments(self, parser):
+        parser.add_argument("state")
+        # parser.add_argument("sessions", nargs="*")
```

Comment:
re-add?

Reply:
@jamesturk Never had it in. It's a leftover from copying over base command info from looking at base_export. Would you like me to add it?